### PR TITLE
focus input after emoji and attachment selection

### PR DESCRIPTION
**Pull request by ambaldwin**
# Submit a pull request

### 🎯 Goal

Focus input after emoji and attachment selection. Close emoji selector upon selection if closeEmojiPickerOnClick prop is true.

### 🛠 Implementation details

Update useEmojiPicker's onSelectEmoji method and useAttachment's uploadNewFiles method to focus. Add `closeEmojiPickerOnClick` prop to `MessageInput` to optionally close the picker on select.

### CODE REVIEW

File: src/components/MessageInput/hooks/useAttachments.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```
Comment (zacheryconverse) [inline src/components/MessageInput/hooks/useAttachments.ts:1]: Did you mean to leave this disable in?

File: src/components/MessageInput/hooks/useFileUploads.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ export const useFileUploads = <
   const { t } = useTranslationContext('useFileUploads');
 
   const uploadFile = useCallback((id) => {
+    console.log('uploadFile');
```
Comment (zacheryconverse) [inline src/components/MessageInput/hooks/useFileUploads.ts:48]: this one, as well

Comment (ambaldwin) [reply src/components/MessageInput/hooks/useFileUploads.ts:48]: I'll take this out.

File: src/components/MessageInput/hooks/useAttachments.ts (hunk 1 above)
Comment (ambaldwin) [reply src/components/MessageInput/hooks/useAttachments.ts:1]: I'm not sure how this was added, I'll remove.

**Comment by petyosi** [comment]
@ambaldwin I tested this locally, spotted two things:

- the picker does not close itself (can repro that if you want)
- If I pick an emoji from the thread composer, it focuses the input in the main one.

**Comment by ambaldwin** [comment]
- @petyosi did you add the prop `closeEmojiPickerOnClick` to the `MessageInput` for the picker close?
- Working on correct focus on input.

**Comment by petyosi** [comment]
> * @petyosi did you add the prop `closeEmojiPickerOnClick` to the `MessageInput` for the picker close?
I did not, my mistake. Should be in the docs of the prop if not already.

**Comment by ambaldwin** [comment]
Ok @petyosi -ready for review again. thanks!

**Review by petyosi — APPROVED** [review]

**Comment by ambaldwin** [comment]
chore: add closeEmojiPickerOnClick and focus input

**Pull request merged**